test_vgpu.sh

```diff
@@ -77,6 +102,15 @@ _SSH_OPTIONS="\
     -o UserKnownHostsFile=/dev/null \
     -i /opt/stack/.ssh/id_rsa"
 
+pushd $ROOT_DIR
+rm $DEVSTACK_PATH -rf
+git clone https://github.com/openstack-dev/devstack.git
+
+sed -i '/iniset $NOVA_CONF DEFAULT flat_injected \"False\"/a\    iniset $NOVA_CONF xenserver image_handler "$vdi_remote_stream"' $DEVSTACK_PATH/lib/nova_plugins/hypervisor-xenserver
```

Comment:
This is quite ugly... I thought there was a way to do this with hook scripts, or in the vGPU job config on Jenkins rather than in this script?

Reply:
Will check if we can do it.